be/src/olap/olap_define.h

```diff
@@ -53,7 +53,10 @@ static constexpr uint32_t OLAP_COMPACTION_DEFAULT_CANDIDATE_SIZE = 10;
 // the max length supported for varchar type
 static const uint16_t OLAP_STRING_MAX_LENGTH = 65535;
 
-static const int32_t PREFERRED_SNAPSHOT_VERSION = 3;
+static const int32_t PREFERRED_SNAPSHOT_VERSION = 4;
```

Comment:
should define 4 as BETA_ROWSET_VERSION
and let PREFERRED_SNAPSHOT_VERSION equal BETA_ROWSET_VERSION

Reply:
ok